autogen/main.tf.tmpl

```diff
@@ -79,12 +79,23 @@ resource "google_compute_ssl_certificate" "default" {
   }
 }
 
+resource "google_compute_managed_ssl_certificate" "default" {
+  provider    = google-beta
+  project     = var.project
+  count       = var.ssl && var.managed_ssl_certificate && ! var.use_ssl_certificates ? 1 : 0
```

Comment:
Can't we use `length(var.managed_ssl_certificate_domains) > 0` instead of needing another boolean `var. managed_ssl_certificate`?

Reply:
@bharathkkb absolutely will do. I suspect the same could be done to eliminate the `use_ssl_certificates` in the first place.

Reply:
It seems #97 also suggested `use_ssl_certificates` is superfluous.
I’ll let maintainers decide if they want to remove that (a breaking change).